src/bootstrap/cc_detect.rs

```diff
@@ -132,7 +132,7 @@ pub fn find(build: &mut Build) {
             false
         };
 
-        if cxx_configured {
+        if cxx_configured || target.contains("vxworks") {
```

Comment:
Why is vxworks special here? I would expect the config.toml to contain a cxx entry instead of this.

Reply:
This is to let the linker function in lib.rs can find the compiler for c++.